test/main.js

```diff
@@ -48,4 +57,86 @@ plugins.forEach((plugin) => {
   test(`Plugin name should start with an uppercase letter: ${packageName}`, (t) => {
     t.true(typeof name === 'string' && name === upperCaseFirst(name));
   });
+
+  if (compatibility !== undefined) {
```

Comment:
What are your thoughts on returning early here instead? E.g.

```js
if (compatibility === undefined) return
```

Given all the nesting that happens below I feel like it helps with readability, but this might be a personal preference 😅

Reply:
Good call :+1: 
I just fixed it with early exits.